### ISSUE TITLE
NPE in 2.1.1

### ISSUE THREAD

**Issue by thebridsk** [swagger-akka-http/swagger-scala-module#50]
swagger-akka-http 2.1.0
swagger-scala-module 2.1.1

````
java.lang.NullPointerException
	at com.github.swagger.scala.converter.SwaggerScalaModelConverter.$anonfun$resolve$1(SwaggerScalaModelConverter.scala:50)
	at scala.Option.getOrElse(Option.scala:201)
	at com.github.swagger.scala.converter.SwaggerScalaModelConverter.resolve(SwaggerScalaModelConverter.scala:35)
	at io.swagger.v3.core.converter.ModelConverterContextImpl.resolve(ModelConverterContextImpl.java:90)
	at io.swagger.v3.core.jackson.ModelResolver.resolve(ModelResolver.java:655)
	at com.github.swagger.scala.converter.SwaggerScalaModelConverter.$anonfun$resolve$1(SwaggerScalaModelConverter.scala:44)
	at scala.Option.getOrElse(Option.scala:201)
	at com.github.swagger.scala.converter.SwaggerScalaModelConverter.resolve(SwaggerScalaModelConverter.scala:35)
	at io.swagger.v3.core.converter.ModelConverterContextImpl.resolve(ModelConverterContextImpl.java:90)
	at io.swagger.v3.core.converter.ModelConverters.readAllAsResolvedSchema(ModelConverters.java:99)
	at io.swagger.v3.core.util.AnnotationsUtils.resolveSchemaFromType(AnnotationsUtils.java:562)
	at io.swagger.v3.core.util.AnnotationsUtils.getSchema(AnnotationsUtils.java:1126)
	at io.swagger.v3.core.util.AnnotationsUtils.getSchema(AnnotationsUtils.java:1116)
	at io.swagger.v3.core.util.AnnotationsUtils.getContent(AnnotationsUtils.java:1068)
	at io.swagger.v3.jaxrs2.OperationParser.getApiResponses(OperationParser.java:91)
	at io.swagger.v3.jaxrs2.Reader.setOperationObjectFromApiOperationAnnotation(Reader.java:1184)
	at io.swagger.v3.jaxrs2.Reader.parseMethod(Reader.java:960)
	at io.swagger.v3.jaxrs2.Reader.parseMethod(Reader.java:818)
	at io.swagger.v3.jaxrs2.Reader.read(Reader.java:450)
	at io.swagger.v3.jaxrs2.Reader.read(Reader.java:169)
	at com.github.swagger.akka.SwaggerGenerator.filteredSwagger(SwaggerHttpService.scala:116)
	at com.github.swagger.akka.SwaggerGenerator.filteredSwagger$(SwaggerHttpService.scala:115)

````

Lines of interest:

              val required = property.getRequired
              required.remove("traversableAgain")
              required.remove("empty")

NPE is at the remove traversableAgain line.  Looks like property.getRequired can return a null.

**Comment by pjfanning** [comment]
Thanks for reporting this. I've committed a fix and will look at doing a 2.1.2 release soon

**Comment by pjfanning** [comment]
fixed in v2.1.2
